software/contrib/pams.py

```diff
@@ -1397,23 +1385,17 @@ def main(self):
             for cv in CV_INS.values():
                 cv.update()
 
-            elapsed_time = time.ticks_diff(now, self.last_interaction_time)
-            if elapsed_time > BLANK_TIMEOUT_MS:
-                self.screensaver.draw_blank()
-            elif elapsed_time > SCREENSAVER_TIMEOUT_MS:
-                self.screensaver.draw()
-            else:
-                oled.fill(0)
-                self.main_menu.draw()
+            ssoled.fill(0)
```

Comment:
might as well use .clear() to show off the extrra function of ssoled!

Reply:
`.clear()` was removed; it shouldn't have existed in the first place.